components/style/properties/data.py

```diff
@@ -75,7 +75,8 @@ def __init__(self, name, values, gecko_constant_prefix=None,
         self.extra_gecko_aliases = parse_aliases(extra_gecko_aliases or "")
         self.extra_servo_aliases = parse_aliases(extra_servo_aliases or "")
         self.consts_map = {} if custom_consts is None else custom_consts
-        self.gecko_strip_moz_prefix = gecko_strip_moz_prefix
+        self.gecko_strip_moz_prefix = True \
```

Comment:
Why did you need to change this? You pass `False` in the definition of `-moz-user-select` so why is this needed?